app/src/main/java/com/beemdevelopment/aegis/ui/fragments/preferences/BackupsPreferencesFragment.java

```diff
@@ -221,6 +244,15 @@ private CharSequence getBackupStatusMessage(@Nullable Preferences.BackupResult r
         return spannable;
     }
 
+    private void createBackupFile() {
+        String fileName = VaultBackupManager.FILENAME_PREFIX +".json";
```

Comment:
Please add the following below the definition of VaultBackupManager.FILENAME_PREFIX and refer to it here:

```java
public static final String FILENAME_SINGLE = String.format("%s.json", FILENAME_PREFIX);
```